src/lib/style-properties.html

```diff
@@ -78,18 +78,13 @@
 
       // returns cssText of properties that consume variables/mixins
       collectCssText: function(rule) {
-        var customCssText = '';
         var cssText = rule.parsedCssText;
         // NOTE: we support consumption inside mixin assignment
         // but not production, so strip out {...}
         cssText = cssText.replace(this.rx.BRACKETED, '')
           .replace(this.rx.VAR_ASSIGN, '');
         var parts = cssText.split(';');
-        for (var i=0, p; i<parts.length; i++) {
-          p = parts[i];
-          customCssText += p + ';\n';

```

Comment:
This ad-hoc join of the parts was causing a vestigial trailing semi-colon to be introduced in cases where the last item in the `parts` list is an empty string.